utils/snowplow_split_events_model_gen.py

```diff
@@ -353,9 +172,18 @@ def validate_json(jsonData: dict, schema: dict = None, validate: bool = True) ->
     # Write model string
     model_content = f"""{{{{ config(
     tags = "snowplow_web_incremental",
-    materialized = var("snowplow__incremental_materialization"),
+    materialized = var("snowplow__incremental_materialization", "snowplow_incremental"),
     unique_key = "event_id",
-    upsert_date_key = "collector_tstamp"
+    upsert_date_key = "collector_tstamp",
+    partition_by = snowplow_utils.get_partition_by(bigquery_partition_by={{
+      "field": "collector_tstamp",
+      "data_type": "timestamp"
+    }}, databricks_partition_by='collector_tstamp'),
```

Comment:
Here you want to partition by `collector_tstamp_date` rather than `collector_tstamp`, as Databricks is not smart about partitioning by timestamps and will instead partition for every individual distinct timestamp rather than aggregating up to a daily level as you might expect